### ISSUE TITLE
Argument named `ignore` triggers redifinition warning

### ISSUE THREAD

**Issue by mauskin** [douglascrockford/JSLint#260]
When in a callback only two out of four arguments are used the second argument named `ignore` triggers a warning “Redefinition of 'ignore' from line X.”

It happend while using a library that takes that kind of callback and in my code I need to use only the second and the fourth argument.

```js
function (ignore, second, ignore, fourth) {
    doSomething(second);
    doSomethingElse(fourth);
}
```

Shouldn’t `ignore` argument be ignored for the redifinition check also?

**Comment by douglascrockford** [comment]
That would be a syntax error.

**Comment by fourier-space** [comment]
Can we have multiple explicit ignored parameters?
```
function (ignore, second, ignore2, fourth) {
    doSomething(second);
    doSomethingElse(fourth);
}
```
Where any parameter with a name starting with "ignore" is ignored.

**Comment by douglascrockford** [comment]
Complain to the maker of the function that it takes too many unnecessary arguments.
